docs/site_pipeline.rst

```diff
@@ -177,6 +177,104 @@ including amplitude and FWHM.
    :module: sotodlib.site_pipeline.analyze_bright_ptsrc
    :func: get_parser
 
+finalize-focal-plane
+--------------------
+
+This element produces a finalized focal plane for a given array.
+It consumes the output of ``make_position_match`` and is designed to
+conbine results across multiple tuning epochs.
+
+.. automodule:: sotodlib.site_pipeline.finalize_focal_plane
+   :members:
+   :undoc-members:
+
+
+Config file format
+``````````````````
+
+Here's an annotated example:
+
+.. code-block:: yaml
+
+  # Data sources
+  context:
+      path: "context.yaml"
+      position_match: "position_match"
+      obs_ids:
+          - "obs_ufm_uv8_1678148785"
+      query: "query_string"
+
+  # For multi observation results just specify the path
+  multi_obs:
+    - "path_to_h5"
+  # These should be one per provided result
+  # If a given has context setup to load this information set the value to null
+  # Order is the ones from obs_id first, then query, then multi_obs
+  detmaps:
+      - null
+      - "path_to_detmap"
+  
+  # Configuration options
+  use_matched: False # Set to True to use the mapping from make_position_match
```

Comment:
Please explain what use_matched does in more detail (perhaps not in the comments but rather in the main text of the help).  I.e. it takes some important parameter (the det_id) from X instead of from Y.